test_runner/regress/test_timeline_delete.py

```diff
@@ -346,20 +346,31 @@ def test_concurrent_timeline_delete_if_first_stuck_at_index_upload(
     failpoint_name = "persist_index_part_with_deleted_flag_after_set_before_upload_pause"
```

Comment:
The comment above this line is outdated.
Can't make a suggestion because GitHub limitations.

```
# Make uploads stop after setting the deleted_at flag locally but before uploading the IndexPart.
# We'll make two delete_timeline API calls after this.
# The first one will get stuck at the failpoint.
# The second one will coalesce with the first call, i.e., it will wait for it to finish.
# We'll unstuck the first call, and verify both calls report success.
```

Reply:
I'll just remove this comment and fix the test function doc.

Reply:
6b1adf6e